src/main/java/com/crud/tasks/controller/TrelloController.java

```diff
@@ -8,21 +8,26 @@
 import org.springframework.web.bind.annotation.RestController;
 
 import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
 
-    @RestController
+@RestController
     @RequestMapping("/v1/trello")
     public class TrelloController {
 
         @Autowired
         private TrelloClient trelloClient;
 
         @RequestMapping(method = RequestMethod.GET, value = "getTrelloBoards")
-        public void getTrelloBoards() {
+        public Optional<List<TrelloBoardDto>> getTrelloBoards() {
```

Comment:
Metoda nigdy nie powinna zwracac Optionala kolejcji. To antypattern. Jesli elementow nie ma wcale, to powinna byc zwrócona pusta lista